src/cli/cli_tcp.cpp

```diff
@@ -494,6 +637,18 @@ template <> otError TcpExample::Process<Cmd("sendend")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli tcp abort
+ * @code
+ * tcp abort
+ * TCP: Connection reset
+ * Done
+ * @endcode
+ * @par
+ * Unceremoniously ends the TCP connection, if one exists, associated with the
+ * example TCP endpoint, transitioning the TCP endpoint to the closed state.
```

Comment:
Do we need "if one exists"?  What happens if this command is used and there's no connection?

We probably can drop that clause since it should be implied (we don't say that for other TCP commands):

"Unceremoniously ends the TCP connection associated with the example TCP endpoint, transitioning the TCP endpoint to the closed state."